drivers/display/display_mcux_dcnano_lcdif.c

```diff
@@ -290,14 +332,10 @@ static DEVICE_API(display, mcux_dcnano_lcdif_api) = {
 	}									\
 	MCUX_DCNANO_LCDIF_FRAMEBUFFER_DECL(n);					\
 	struct mcux_dcnano_lcdif_data mcux_dcnano_lcdif_data_##n = {		\
-		.fb_config = {							\
-			.enable = true,						\
-			.enableGamma = false,					\
-			.format = DT_INST_PROP(n, pixel_format),		\
-		},								\
-		.next_idx = 0,							\
-		.pixel_bytes = MCUX_DCNANO_LCDIF_PIXEL_BYTES(n),		\
-	};									\
+		MCUX_DCNANO_LCDIF_FB_CONFIG(n)									\
```

Comment:
Same request here- using `git diff` can often make it easier to spot this alignment issue with the `\`, if your editor isn't helping.